### ISSUE TITLE
Failed to load 'magit' and 'org' package incrementally

### ISSUE THREAD

**Issue by sbacelar** [hlissner/doom-emacs#3394]
**What actually happened?**
When I open `doom-emacs` I receive two similar error messages:

`Failed to load 'magit' package incrementally because (user-error The Org-roam database was created with a newer Org-roam version)`
and
`Failed to load 'org' package incrementally because (user-error The Org-roam database was created with a newer Org-roam version)`

**System information:**
<details><pre>
((emacs
  (version . "26.3")
  (features . "XPM JPEG TIFF GIF PNG RSVG SOUND NOTIFY ACL GNUTLS LIBXML2 ZLIB TOOLKIT_SCROLL_BARS THREADS LCMS2")
  (build . "Aug 29, 2019")
  (buildopts "--without-dbus --host=x86_64-w64-mingw32 --without-compress-install 'CFLAGS=-O2 -static -g3'")
  (windowsys . batch)
  (daemonp . server-running))
 (doom
  (version . "2.0.9")
  (build . "HEAD -> develop a262527a7 2020-05-21 04:33:39 -0400")
  (dir . "~/.doom.d/"))
 (system
  (type . windows-nt)
  (config . "x86_64-w64-mingw32")
  (shell . "C:/Program Files/emacs-26.3-x86_64/libexec/emacs/26.3/x86_64-w64-mingw32/cmdproxy.exe")
  (uname . "n/a")
  (path "c:/Program Files/PowerShell/7-preview" "C:/Python38/Scripts/" "C:/Python38/" "C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v10.2/bin" "C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v10.2/libnvvp" "C:/Windows/system32" "C:/Windows" "C:/Windows/System32/Wbem" "C:/Windows/System32/WindowsPowerShell/v1.0/" "C:/Windows/System32/OpenSSH/" "C:/Program Files/Git/cmd" "C:/ProgramData/chocolatey/bin" "C:/Program Files/emacs-26.3-x86_64/bin" "C:/Program Files (x86)/Calibre2/" "C:/Program Files/NVIDIA Corporation/Nsight Compute 2019.5.0/" "C:/Program Files (x86)/NVIDIA Corporation/PhysX/Common" "C:/Program Files/NVIDIA Corporation/NVIDIA NvDLISR" "~/AppData/Local/Programs/Julia/Julia-1.4.1/bin" "C:/Program Files (x86)/Vim/vim82" "C:/Program Files/Calibre2/" "~/.emacs.d/bin" "C:/Program Files/PowerShell/7-preview/preview" "C:/Program Files/sqlite3" "~/scoop/apps/graphviz/2.38/bin" "~/scoop/apps/gcc/current/bin" "~/scoop/shims" "~/AppData/Local/Microsoft/WindowsApps" "~/AppData/Local/Programs/Microsoft VS Code/bin" "~/AppData/Local/Programs/MiKTeX 2.9/miktex/bin/x64/" "~/AppData/Local/Julia-1.3.1/bin" "C:/Program Files/wkhtmltopdf/bin" "~/Anaconda3" "~/Anaconda3/Scripts" "~/AppData/Local/atom/bin" "~/AppData/Local/Pandoc/" "~/CMD" "C:/Program Files (x86)/GnuWin32/bin" "c:/Program Files/emacs-26.3-x86_64/libexec/emacs/26.3/x86_64-w64-mingw32"))
 (config
  (envfile . envvar-file)
  (elc-files . 0)
  (modules :completion company helm ivy :ui deft doom doom-dashboard doom-quit hl-todo modeline ophints (popup +defaults) unicode vc-gutter vi-tilde-fringe workspaces :editor (evil +everywhere) file-templates fold snippets :emacs dired electric undo vc :term term :checkers syntax spell grammar :tools (eval +overlay) lookup magit :lang emacs-lisp julia latex markdown (org +roam) python sh :config (default +bindings +smartparens))
  (packages (emacsql-sqlite3) (org-ref) (org-roam-bibtex))
  (unpin "n/a")
  (elpa "n/a")))
</pre></details>

**Comment by leoalekseyev** [comment]
Disabling roam in your `doom.d/init.el` will fix this. 
Assuming you can rebuild your org-roam db (or don't care if it gets wiped), deleting the database file should also do the trick.

**Comment by sbacelar** [comment]
@leoalekseyev And after that, shall I be able to reinstall `org-roam` again without problem? I simply followed the instructions to install it in the init file (`org +roam`). I don't understand why the error message says "it was created with a newer Org-roam version".

**Comment by sbacelar** [comment]
Solved! Both in Windows and MacOS with `(unpin! org-roam)` and after that `doom update`.

**Comment by chrislockard** [comment]
I ran into a similar issue. All I added was `+roam` to my `init.el` and after a `doom up` I was greeted with
`Failed to load org package incrementally, because: (error Selecting deleted buffer)`. If I remove the `+roam` tag from lang>org and run `doom up` the problem is fixed. However, I can't use org-roam :(

<details>
<summary>System Info</summary>
<br>
```
((emacs
  (version . "27.0.91")
  (features . "RSVG IMAGEMAGICK DBUS GLIB NOTIFY KQUEUE ACL GNUTLS LIBXML2 ZLIB TOOLKIT_SCROLL_BARS NS MODULES THREADS XWIDGETS JSON PDUMPER LCMS2 GMP")
  (build . "May 15, 2020")
  (buildopts "--disable-dependency-tracking --disable-silent-rules --enable-locallisppath=/usr/local/share/emacs/site-lisp --infodir=/usr/local/Cellar/emacs-plus/HEAD-e75f6be/share/info/emacs --prefix=/usr/local/Cellar/emacs-plus/HEAD-e75f6be --with-xml2 --with-gnutls --with-dbus --with-imagemagick --with-json --with-modules --with-rsvg --without-pop --with-xwidgets --with-ns --disable-ns-self-contained")
  (windowsys . batch)
  (daemonp . server-running))
 (doom
  (version . "2.0.9")
  (build . "HEAD -> develop 847bbf380 2020-06-13 17:12:23 -0400")
  (dir . "~/.config/dotfiles/Emacs/.doom.d/"))
 (system
  (type . darwin)
  (config . "x86_64-apple-darwin19.4.0")
  (shell . "/usr/local/bin/zsh")
  (uname . "Darwin 19.5.0 Darwin Kernel Version 19.5.0: Tue May 26 20:41:44 PDT 2020; root:xnu-6153.121.2~2/RELEASE_X86_64 x86_64")
  (path "/usr/local/opt/python@3.8/bin" "~/.cargo/bin" "/usr/local/bin" "/usr/bin" "/bin" "/usr/sbin" "/sbin" "/Applications/Wireshark.app/Contents/MacOS" "/usr/local/opt/python@3.8/bin" "~/.emacs.d/bin" "/usr/local/bin" "~/.emacs.d/bin" "/usr/local/Cellar/emacs-plus/HEAD-e75f6be/libexec/emacs/27.0.91/x86_64-apple-darwin19.4.0"))
 (config
  (envfile . envvar-file)
  (elc-files . 0)
  (modules :completion company (ivy +icons +fuzzy) :ui deft doom doom-dashboard doom-quit fill-column hl-todo modeline nav-flash ophints (popup +all +defaults) (pretty-code +fira) tabs treemacs unicode vc-gutter vi-tilde-fringe window-select workspaces minimap :editor (evil +everywhere) fold format rotate-text snippets :emacs (dired +icons) electric (ibuffer +icons) (undo +tree) vc :term term :checkers (syntax +inline) (spell +inline) :tools editorconfig (eval +overlay) (lookup +docsets) lsp macos (magit +forge) rgb :lang common-lisp data emacs-lisp markdown (org +dragndrop +hugo +ipython +mu4e +pomodoro +pandoc +present +roam) (python +lsp) (rust +lsp) sh web :email mu4e :config literate (default +bindings +smartparens))
  (packages (nyan-mode) (2048-game) (easy-hugo) (org-jira))
  (unpin "n/a")
  (elpa "n/a")))
```
</details>

**Comment by sbacelar** [comment]
First `+roam` on `init.el`. Then `unpin! org-roam` in `packages.el`. And finally `doom update`.
That's what I did and it works.

**Comment by chrislockard** [comment]
@sbacelar I followed your instructions and experience the same issue. Also, trying to load magit with `SPC g g` returns `Selecting deleted buffer` and magit won't open.

The only fix I've found is to remove the `+roam` tag then `doom sync` `doom update`

Notice my error is `Selecting deleted buffer` instead of `user-error The Org-roam database was created with a newer Org-roam version` Perhaps this warrants a new issue?
